src/Tickets/Seating/Commerce/Controller.php

```diff
@@ -137,6 +137,13 @@ public function sync_seated_tickets_stock( $meta_id, $object_id, $meta_key, $met
 			return;
 		}
 
+		$stock = (int) $meta_value;
+
+		if ( 0 > $stock ) {
+			// We are not syncing bugs. Seats can be infinite.
```

Comment:
I think you meant Seats can NOT be infinite?

Reply:
Also, we should move this after the seat type check below to avoid interference with regular tickets.

Reply:
Yes i meant not infinite

Reply:
For the other comment, i think being above the seat check is better. It's a faster check so it should come before slower checks. It can quickly tell us that its not about seating if its `-1`

Reply:
@dpanta94 you are typecasting the value so it may not be the original one that is now returned. There are tests failing for this.